tests/AppTest.php

```diff
@@ -219,8 +218,10 @@ public function testHttpFlowEventNotInstanceOfEventInterface()
         $response = $app->run($request, $response);
     }
 
-    public function testRouteInfoNotInstanceOfRouteInfoInterface()
+    public function testDispatcherNotInstanceOfPennyDispatcher()
     {
+        $this->setExpectedException('RuntimeException', 'Dispatcher must be an instance of Penny\Dispatcher class');
```

Comment:
this test WAS not to tests dispatcher, but for test RouteInfo Not InstanceOf RouteInfoInterface, that's why it WAS silent